requirements.txt

```diff
@@ -1,8 +1,10 @@
 batinfo==0.3
+-e git+https://github.com/bochecha/dbus-python.git@pipified-1.1.1#egg=dbus
```

Comment:
Thanks, that's clear now :)
But let's then move the fork on ideascube org :)

> (I'll note that this was explained in the commit message, did you found it unclear?)

Ooops, sorry, missed that: I'm not looking at each commit when reviewing a PR ;)

> I could sure get that merged back into posborne's fork, but what's the point, if we move to something else than dbus-python anyway?

I'd submit a PR anyway if it's not too much work, we are never sure what's the future will be ;)

> As for getting it on pypi, we would then use the "dbus-python" namespace for a fork, taking it away from the upstream maintainer.

You mean it's already on pypi even without setup.py? o_O